CryptoAnalysis/src/main/java/crypto/exceptions/CryptoAnalysisException.java

```diff
@@ -0,0 +1,10 @@
+package crypto.exceptions;
+
+public class CryptoAnalysisException extends RuntimeException {
+	
+	private static final long serialVersionUID = 1L;
```

Comment:
I'm not an expert here... Is this value OK(ish)?

Reply:
This value is OK. It is generated automatically when creating the exception class and the field is used during serialization and deserialization.